### PR TITLE
96976 Update alert on choose a VA location page - Part 4

### PR DESCRIPTION
**Pull request by vbahinwillit**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**:
   1. First search `vets-website` for _all_ instances of the `entryName` in your `manifest.json` and remove them in a separate PR. Look particularly for references in `src/applications/static-pages/static-pages-entry.js` and `src/platform/forms/constants.js`. _**If you do not do this, other applications will break!**_
      - _Add the link to your merged vets-website PR here_
   2. Then, Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
      - _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary
This PR updates the warning messages content.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#96976
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#0000
- _Link to epic if not included in ticket_
department-of-veterans-affairs/va.gov-team#0000

## Testing done
Unit and e2e testing.

## How to test locally
You can toggle the "enabled" attribute for a given service in the "scheduling_configuration.json" file to enable/disable request or direct schedule appointments for a location. Ex.

```
{
  "data": [
    {
      "id": "983",
      "type": "scheduling_configuration",
      "attributes": {
        "facilityId": "983",
        "services": [
          {
            "id": "amputation",
            "name": "Amputation Services",
            "stopCodes": [{ "primary": "211" }],
            "direct": {
              "patientHistoryRequired": true,
              "patientHistoryDuration": 1095,
              "canCancel": true,
              "enabled": true
            },
            "request": { "enabled": false }
          }
        ]
    } 
    ]
}
```

Add the following code snippet to "./vaos/services/mocks/index.js" to simulate a 'patient-history-insufficient' error:

```
    if (
      isDirect &&
      req.query.facility_id.startsWith('983') &&
      req.query.clinical_service_id === 'amputation'
    ) {
      ineligibilityReasons.push({
        coding: [
          {
            code: 'patient-history-insufficient',
          },
        ],
      });
    }
```
Toggle the "isDirect" variable in the 'if' statement to mock the response for direct/request appointment requests. Ex.

**Request** - if(!isDirect && req.query.facility_id.startsWith('983') && req.query.clinical_service_id === 'amputation')
**Direct** - if(isDirect && req.query.facility_id.startsWith('983') && req.query.clinical_service_id === 'amputation')

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  | localhost_3001_my-health_appointments_schedule_location(iPhone SE) (1) | localhost_3001_my-health_appointments_schedule_location(iPhone SE) (3) |
| Desktop |  localhost_3001_my-health_appointments_schedule_location (3) | localhost_3001_my-health_appointments_schedule_location (5) |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

## Acceptance criteria
VAOS

### Quality Assurance & Testing
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js [hunk 1]
```diff
@@ -89,20 +85,27 @@ export default function getEligibilityMessage({
...
     );
   } else if (requestReason === ELIGIBILITY_REASONS.noRecentVisit) {
-    const monthRequirement = settings?.request?.patientHistoryDuration
-      ? (settings.request.patientHistoryDuration / 365) * 12
-      : '24';
-    title = 'We can’t find a recent appointment for you';
+    const contact = facilityDetails?.telecom?.find(
+      tele => tele.system === 'phone',
+    )?.value;
+    title = 'We couldn’t find a recent appointment for you';
```
Comment (JunTaoLuo) [inline src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js:91]: Hmm is this the right block to modify? The linked ticket mentioned:
>Scenario:
> - DS is enabled
> - Facility requires a previous encounter
> - AND User has no previous encounters
> - AND Requests are disabled (VATS: notSupported)

Which sounds like the block from lines 18-46 should be changed instead.

**Review by JunTaoLuo — CHANGES_REQUESTED** [review]
Hmm this doesn't look right. When I tested this locally, I see:
image

File: src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/new-appointment/components/VAFacilityPage/getEligibilityMessage.js:91]: Response from Melissa: 

Use this title for both: "You can't schedule an appointment online at this facility"

So, closing this PR.

**Comment by melissa-lefevre** [comment]
Looks great, no issues. Thanks!

**Pull request closed without merging**